functions/notification.js

```diff
@@ -0,0 +1,20 @@
+const admin = require("firebase-admin");
+
+module.exports = {
```

Comment:
Really smart to turn this into a function so it can be reused for every notification sent!